modules/ROOT/pages/datasense-concept.adoc

```diff
@@ -8,27 +8,27 @@ DataSense is a feature of Anypoint Studio that uses Mule event metadata to help
 
-Imagine that you are obtaining status update data from the Slack connector and you want to log the message you are sending to Slack, but you aren't familiar with the property names used by this connector. Instead of referring to Slack documentation, you can insert a Logger message processor after the Slack connector and begin to type in it, which activates autocomplete. For example, if you type `#[payload.` and then press `ctrl + space bar`, you get a list of all the properties and methods associated with that payload, including the properties returned by your request to the Slack connector.
+Imagine that you're obtaining status update data from the Slack connector. You want to log the message you're sending to Slack, but you aren't familiar with the property names used by this connector. Instead of referring to Slack documentation, you can insert a Logger message processor after the Slack connector and begin to type in it, which activates autocomplete. For example, if you type `#[payload.` and press `ctrl + space bar`, you get a list of all the properties and methods associated with that payload. In the list, you also get the properties returned by your request to the Slack connector.
 
 image::payload-autocomplete.png[]
 
 == When to Manually Define Metadata
 
-Although all components of a Mule application allow you to manually define the metadata that they expose, you should do so only when it is absolutely required (for example, for a metadata type that you defined), or when it is most cohesive to an annotated value (for example, when a component uses or expects a specific metadata type).
+You can manually define the metadata that all the components of a Mule application expose. You must do this if required (for example, for a metadata type that you defined). Or you must define metadata manually if most cohesive to an annotated value (for example, when a component uses or expects a specific metadata type).
 
-For example, when working with flows and subflows you must declare their metadata contract to specify the shape of the event you require and the shape of the event your return to the caller. On the contrary, you should not define metadata when using the Transform Message component because DataWeave defines the contract between the producer of data and the consumer of data. In this situation, if you explicitly declare a metadata type, you mask the actual metadata type of the data coming from your Transform Message component.
+When working with flows and subflows, you must declare their metadata contract to specify the shape of the event you require and the shape of the event you return to the caller. On the contrary, you must not define metadata when using the Transform Message component because DataWeave defines the contract between the producer of data and the consumer of data. In this situation, if you explicitly declare a metadata type, you mask the actual metadata type of the data coming from your Transform Message component.
```

Comment:
```suggestion
When working with flows and subflows, you must declare their metadata contract to specify the shape of the event you require and the shape of the event you return to the caller. On the contrary, do not define metadata when using the Transform Message component because DataWeave defines the contract between the producer and the consumer of data. In this situation, if you explicitly declare a metadata type, you mask the actual metadata type of the data coming from your Transform Message component.
```

Reply:
OK! Thanks!